mslib/msui/constants.py

```diff
@@ -34,11 +34,12 @@
 # ToDo refactor to generic functions, keep only constants
 HOME = os.path.expanduser(f"~{os.path.sep}")
 MSUI_CONFIG_PATH = os.getenv("MSUI_CONFIG_PATH", os.path.join(HOME, ".config", "msui"))
+_fs = None
 if '://' in MSUI_CONFIG_PATH:
     try:
-        _fs = fs.open_fs(MSUI_CONFIG_PATH)
+        _fs = fs.open_fs(fs.path.dirname(MSUI_CONFIG_PATH))
```

Comment:
look at the idea in the issue. There should be only one attempt for the fs.errors.CreateFailed because it is a remote system, 

and it would be good to have a test for describing the problem and evaluating the fix by that